VideoWeb/VideoWeb/ClientApp/src/app/shared/header-logo-svg/header-logo-svg.component.html

```diff
@@ -0,0 +1,6 @@
+<svg role="presentation" focusable="false" class="moj-header__logotype-crest" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40" height="40" width="40">
```

Comment:
https://css-tricks.com/accessible-svgs/
https://stackoverflow.com/questions/4697100/accessibility-recommended-alt-text-convention-for-svg-and-mathml

Add role="img" and aria-label="somethingHere" to the svg
Add a <title> and <description> inside as first children.

Make sure there is welsh translations for these.